mapstory/views.py

```diff
@@ -895,7 +895,10 @@ def layer_detail(request, layername, template='layers/layer_detail.html'):
         metadata_form = MetadataForm(instance=layer)
         published_form = PublishStatusForm(instance=layer)
 
-    content_moderators = Group.objects.filter(name='content_moderator').first()
+    content_moderator = False
+    for group in request.user.groups.all():
+        if group.name == 'content_moderators':
```

Comment:
Is the content_moderators group added by default? Wondering if it exists in all environments?